share/completions/rcctl.fish

```diff
@@ -0,0 +1,3 @@
+
+complete -c rcctl -xa 'check ls reload restart stop start disable enable' -n 'not __fish_seen_subcommand_from list check ls reload restart stop start enable disable'
+complete -c rcctl -n '__fish_seen_subcommand_from check reload restart stop start enable disable' -xa '(set -l files /etc/rc.d/*; string replace '/etc/rc.d/' '' -- $files)'
```

Comment:
This probably doesn't work properly because of the quoting - remember, you're already inside single quotes, so if you're going to use them you need to escape them (`\'`). Since that confuses some syntax highlighters, I usually use double-quotes in cases like these. I.e.:

```fish
-xa '(set -l files /etc/rc.d/*; string replace "/etc/rc.d/" "" -- $files)'
```